README.md

```diff
@@ -78,6 +78,13 @@ const searchClient = instantMeiliSearch(
 
 InstantMeilisearch offers some options you can set to further fit your needs.
 
+### Customization
+
+- [`placeholderSearch`]((#placeholder-search)): Enable or disable placeholder search (default: `true`).
```

Comment:
You have double parenthesis, this breaks the link :D

Reply:
Oh thanks!